objects/call/calls.go

```diff
@@ -21,12 +23,12 @@ type callsCmd struct {
 }
 
 // getMarshalableCall returns a call struct that we can marshal to JSON and output
-func getMarshalableCall(call *models.Call) interface{} {
+func getMarshalableCall(call *modelsv2.Call) interface{} {
 	if call.Error != "" {
 		return struct {
 			ID          string          `json:"id"`
 			AppID       string          `json:"appId"`
-			Path        string          `json:"path"`
+			FnID        string          `json:"fnId"`
```

Comment:
all of our apis in fn are snake case... but i'll punt ;)

Reply:
`appId` was already like that and I don't want to change this now